app/views/shared/_entries.erb

```diff
@@ -0,0 +1,42 @@
+<% @solutions.each do |solution, user, position| %>
```

Comment:
if we're renaming to submissions, rename the template as well to _submissions.erb?

Reply:
done